src/aria/widgets/form/list/CfgBeans.js

```diff
@@ -99,6 +99,15 @@ module.exports = Aria.beanDefinitions({
                     $description : "strict: for strict highlighting (exact match only), always: for selecting the first item everytime, none: for no highlighting",
                     $enumValues : ["strict", "always", "none"],
                     $default : "none"
+                },
+                "waiAria" : {
+                    $type : "json:Boolean",
+                    $description : "If true, accessibility-related DOM attributes are enabled, to comply with WAI-ARIA specifications. This allows screen readers and other accessibility tools to work better."
+                },
+                "listItemDomIdPrefix" : {
```

Comment:
@divdavem Do you really think this is needed to open such an api ? we could figure out something automagically....